### PR TITLE
refactor: unify app interface

### PR DESCRIPTION
**Pull request by hollandjg**
Make all functions callable using:
```
fsdproc load
fsdproc process
fsdproc process-batch -c config.toml
```

### CODE REVIEW

File: flow.cylc [hunk 1]
```diff
@@ -44,19 +45,18 @@
         script = """
             mkdir -p "$IMGDIR"
             BBOX=$($PYTHON -c "import pandas; print(','.join(str(s) for s in list(pandas.read_csv('${CYLC_WORKFLOW_RUN_DIR}/locations.csv', index_col='location').loc['${LOCATION}'][['left_x', 'lower_y', 'right_x', 'top_y']])))")
-            ${PYTHON} -m ebfloeseg.load $IMGDIR/tci.tiff --debug --kind truecolor --datetime ${DATE} --bbox $BBOX
-            ${PYTHON} -m ebfloeseg.load $IMGDIR/cld.tiff --debug --kind cloud --datetime ${DATE} --bbox $BBOX
-            ${PYTHON} -m ebfloeseg.load $IMGDIR/lnd.tiff --debug --kind landmask --datetime ${DATE} --bbox $BBOX
+            ${FSDPROC} load $IMGDIR/tci.tiff --kind truecolor --datetime ${DATE} --bbox $BBOX
```
Comment (cpaniaguam) [inline flow.cylc:48]: Using the variables looks so much cleaner, thanks!

File: src/ebfloeseg/app.py
Comment (cpaniaguam) [inline src/ebfloeseg/app.py:1]: Consider adding docstrings to these command functions or some comments clarifying what's going on.

File: flow.cylc [hunk 2]
```diff
@@ -44,19 +45,18 @@
         script = """
             mkdir -p "$IMGDIR"
             BBOX=$($PYTHON -c "import pandas; print(','.join(str(s) for s in list(pandas.read_csv('${CYLC_WORKFLOW_RUN_DIR}/locations.csv', index_col='location').loc['${LOCATION}'][['left_x', 'lower_y', 'right_x', 'top_y']])))")
```
[the remark this answers is not in the document]
Comment (hollandjg) [reply flow.cylc:47]: That'll be in #45 ! No problem.

File: src/ebfloeseg/app.py
Comment (hollandjg) [reply src/ebfloeseg/app.py:1]: Makes a lot of sense. I've updated the command help texts and added the citation to the top of the app.

File: src/ebfloeseg/app.py [hunk 3]
```diff
@@ -1,17 +1,141 @@
...
+    wrap: str = "day",
+    kind: ImageType = ImageType.truecolor,
+    bbox: str = "-2334051.0214676396,-414387.78951688844,-1127689.8419350237,757861.8364224486",
+    scale: Annotated[
+        int, typer.Option(help="size of a pixel in units of the bounding box")
+    ] = 250,
+    crs: str = "EPSG:3413",
+    ts: int = 1683675557694,
+    format: str = "image/tiff",
+):
```
Comment (cpaniaguam) [inline src/ebfloeseg/app.py:73]: Use the sample data set here?

File: src/ebfloeseg/app.py [hunk 4]
```diff
@@ -73,8 +191,11 @@ def parse_config_file(config_file: Path) -> ConfigParams:
     return ConfigParams(**defaults)
 
 
-@app.command(name="process-images", help=help, epilog=epilog)
-def process_images(
+@app.command(
+    help="Process a directory of images.",
+    epilog=f"Example: {name} --data-direc /path/to/data --save_figs --save-direc /path/to/save --land /path/to/landfile",
```
Comment (cpaniaguam) [inline src/ebfloeseg/app.py:196]:
```suggestion
    epilog=f"Example: {name} --data-direc /path/to/data --save-figs --save-direc /path/to/save --land /path/to/landfile",
```

**Review by cpaniaguam — CHANGES_REQUESTED** [review]
One observation and one old typo fix here.

File: src/ebfloeseg/app.py (hunk 3 above)
Comment (hollandjg) [reply src/ebfloeseg/app.py:73]: That's in #52

**Review by cpaniaguam — APPROVED** [review]

**Pull request merged**